packages/common/cache/cache.providers.ts

```diff
@@ -17,20 +17,27 @@ export function createCacheManager(): Provider {
       const cacheManager = loadPackage('cache-manager', 'CacheModule', () =>
         require('cache-manager'),
       );
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const cmVersion = require('cache-manager/package.json').version;
+      const cmMajor = cmVersion.split('.')[0];
+      const cachingFactory = (
+        store: CacheManagerOptions['store'],
+        options: Omit<CacheManagerOptions, 'store'>,
+      ): Record<string, any> => {
+        if (cmMajor < 5) {
```

Comment:
The name of the variables would also be fine, I am used to writing them this way, though 😄

```suggestion
      const cacheManagerMajor = cacheManagerVersion.split('.')[0];
      const cachingFactory = (
        store: CacheManagerOptions['store'],
        options: Omit<CacheManagerOptions, 'store'>,
      ): Record<string, any> => {
        if (cacheManagerMajor < 5) {
```

a matter of habit 😄